teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote>Imports should be grouped in the following order:</blockquote><blockquote>- Standard library imports.</blockquote><blockquote>- Related third party imports.</blockquote><blockquote>- Local application/library specific imports.</blockquote><blockquote>You should put a blank line between each group of imports.</blockquote><p>From https://www.python.org/dev/peps/pep-0008/#imports</p><p> </p><p><code>import time</code></p><p><code>from pathlib import Path</code></p><p><code>import requests</code></p><p><code>from urllib.parse import quote</code></p><p> </p><p><code>import pandas as pd</code></p><p><code>from rdkit import Chem</code></p><p><code>from rdkit.Chem import PandasTools</code></p><p><code>from rdkit.Chem.Draw import MolsToGridImage</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>there are two group in this cell, standard library and third party imports, right?</p>

Reply:
<p>I see the formatting in my comment did not work (it's one block), sorry for that. </p><p> </p><p>What I wanted to say in short is: Please move <code>pandas</code>  and <code>requests</code>  to the second import block, since both are third party packages.</p>

Reply:
done